charts/common/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.2.4
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.2.4"
+appVersion: "0.2.5"
```

Comment:
Are there other changes besides adding a `values.yaml` file? And what's the reason for adding the file?

Reply:
Helm-docs wouldn't run for this chart without a values file, hence added it.